### ISSUE TITLE
can you open the file "cigar_cascade100x84_6_stages.xml" ?

### ISSUE THREAD

**Issue by zlszhonglongshen** [Paradiddle131/Cigarette-Detection#1]
this is a good job! can you open the file "cigar_cascade100x84_6_stages.xml" ?

**Comment by trohit920** [comment]
@zlszhonglongshen @Paradiddle131 Please can you share the trained classifier ? If not is there any training data used for training Haar Cascade file?

**Comment by OrkhanHI** [comment]
Can you share the pretrained `cigar_cascade100x84_6_stages.xml` file?

**Comment by Paradiddle131** [comment]
Hi, since it's been years when I first created this GitHub repo, I no
longer have the pre-trained files or other necessary files. However, I
created another cigarette detection repository using Mask RCNN, and some of
the files including images are still available in the drive link below.
Hope this helps.

https://drive.google.com/drive/folders/146Ir_ECB-rbEA9ygO6qzkWhSSA4a0PYC?usp=sharing

Cheers,

Ensar
